### ISSUE TITLE
Codium not detected by github desktop

### ISSUE THREAD

**Issue by goyalyashpal** [desktop/desktop#13295]
copying 🐛 over from https://github.com/VSCodium/vscodium/issues/907

_update:_

* duplicate of: https://github.com/desktop/desktop/issues/10446
* fixed by: https://github.com/desktop/desktop/pull/16511

### Describe the bug
Github desktop (hereafter as ghd) is not recognising VSCodium

### Version & OS

- Ghd: 2.9.4 in "AppData/Local"
- OS: Windows 10 20H2
- Architecture: x64
- Codium: 1.61.2  
from [`VSCodium-x64-updates-disabled-1.61.2.msi`](https://github.com/VSCodium/vscodium/releases/download/1.61.2/VSCodium-x64-updates-disabled-1.61.2.msi) installed in "Program Files" (i.e. i guess for all users)
- NP++: 7.9.5 in "Program Files"
- Typora: 0.9.98 (beta) in "Program Files"

### Steps to reproduce the behavior

1. Open ghd
2. Go to "Files > Options" (<kbd>^</kbd><kbd>,</kbd>)
3. Then go to "Integrations > External Editor"
4. See that VSCodium is not there

### Expected behavior

VSCodium should be there

### Actual behavior

VSCodium is not there

### Screenshots

![image](https://user-images.githubusercontent.com/19423063/140774219-4e0f5d01-4a14-473f-9355-a020e406c6b5.png)

### Logs

Attach your logs by opening the `Help` menu and selecting `Show Logs...`, if applicable.

### Additional context

- I had ghd already installed since last month. I installed VSCodium only last week or so.
- I have restarted pc, and ghd ofc
- vscodium package used downloaded via https://github.com/VSCodium/vscodium/releases/tag/1.61.2
- similar issue: https://github.com/desktop/desktop/issues/7024

**Comment by sergiou87** [comment]
Hi @yashpalgoyal1304 ! 👋 

Could you check in `regedit` if there is anything related to that VSCodium from these paths?

https://github.com/desktop/desktop/blob/da05a90a4b4310a3b98bf878794c0e9fa993f481/docs/technical/editor-integration.md?plain=1#L101-L113

That's where Desktop looks for the info about the integrated editors.

**Comment by goyalyashpal** [comment]
Thanks @sergiou87 for getting back. 

Yes, VSCodium is there under the "64-bit Windows software":

`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{4AA741DE-77C9-4986-9E66-D2D255062D79}`

**Comment by sergiou87** [comment]
Interesting, thank you!! It doesn't have a `_is1` at the end after `}`? Seems like other versions of VSCodium have it.

Anyway, that means we need to add a new entry here: https://github.com/desktop/desktop/blob/da05a90a4b4310a3b98bf878794c0e9fa993f481/app/src/lib/editors/win32.ts#L201-L222

More specifically, something like this:
```typescript
      LocalMachineUninstallKey('{4AA741DE-77C9-4986-9E66-D2D255062D79}'),
```

**Comment by goyalyashpal** [comment]
is there some way to ensure that there is not some version of vscodium with yet another key??

**Comment by sergiou87** [comment]
I don't think so, other than looking for the VSCodium path from `regedit`. You can also check if any of the keys listed in my previous comment is present in your registry 🤔

**Comment by goyalyashpal** [comment]
> You can also check if any of the keys listed in my previous comment is present in your registry 🤔

checked, and none of the keys present in ur prev comment is there in my reg. and no other key is for vscodium in whole of local user registry folder

**Comment by shaneknysh** [comment]
Not sure if this should be a new issue but my VSCodium registry key is different than the one above

`Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{9AF31473-6813-4391-AE4C-E926765F08C3}`

**Comment by safetycar** [comment]
I've been trying to fix this issue for some time.
I was finally able to make it work by using the vscodium's EXE installer instead of the MSI.

**Comment by goyalyashpal** [comment]
> was finally able to make it work by using the vscodium's EXE installer instead of the MSI.

wow, great. i will try .exe installer the next time too. thanks a lot for sharing.

did u lost any settings in vscode by doing that?

**Comment by safetycar** [comment]
If you run the exe you get asked if you want to continue because there's already content in the folder. But I clicked continue and I still see things as they were. Extensions, settings, history... I haven't noticed anything weird.

**Comment by goyalyashpal** [comment]
> I was finally able to make it work by using the vscodium's EXE installer instead of the MSI.

so, i finally did it with `VSCodiumSetup-x64-1.71.2.22258.exe`, and ... it still did not work :(

**Comment by safetycar** [comment]
I did more than one test, uninstalling the EXE and installing through MSI and it was failing when it was MSI.
The EXE leaves an uninstaller "unins001.exe" for me, that the MSI doesn't leave.
I don't remember the details exaclty, I used that uninstaller at some point. I don't know if It was part of what made things work or not... 
But also don't remember losing configs, so I don't know at what point I used the uninstaller :/
